source/versions/versioning.rst

```diff
@@ -104,9 +105,11 @@ A release which has the ``-SNAPSHOT`` label represents the next API version in d
 Release Candidate
 =================
 
-Release Candidates (**RC**) are builds in which bug fixes are released for implementations while they are still in 
-development. They are nearly ready for a full release, but require more bug fixes and testing.
-
+Any push made to GitHub which is not a *Recommended Build* is a *Release Candidate*. Any release candidate can become a 
+recommended build. Furthermore, the code may or may not change; the change might occur in a settings file. The 
+resulting build may run as well as a recommended build, or it may not run at all. In the end, a change occurred in the 
+repository, it triggered a build, and the resulting build is not a recommended build. Therefore, it is a release 
+candidate.
```

Comment:
Brick-a-budgie, that's horrible to read. Accurate but rambling, methinks it has too many words by far. I mean, the only thing we really need to get across is that RC's *might not work properly*, but may become RB's after further testing. Getting into the nuts-and-bolts of the process (e.g. "triggered a build") is unnecessary, imho.